buddyboss/buddyboss-platform
# #321 - Fixed Activity Read More Animation

### All Submissions:


### Changes proposed in this Pull Request:

Closes #321 

### How to test the changes in this Pull Request:

1. Go to Activity Feed
2. Add enough content into a post to see a Read More link in the post, or just test this in the activity feed.
3. Click Read More, and watch the animation.

### Other information:



### Changelog entry

> Changed Read More animation in Activity Feed.
